src/prompt.c

```diff
@@ -127,6 +127,20 @@ prompt_default_handler(struct input *input, struct key *key)
 }
 
 static enum input_status
+prompt_script_handler(struct input *input, struct key *key)
```

Comment:
There might be a smarter way to do it, but the `NULL` handler doesn't recognize newlines.